test/unit/sdam/spec.test.js

```diff
@@ -164,7 +164,12 @@ function cloneForCompare(event) {
 
 function executeSDAMTest(testData, testDone) {
   parse(testData.uri, (err, parsedUri) => {
-    if (err) return done(err);
+    if (err) {
+      if (err.message.toLowerCase().indexOf('load balancer') !== -1) {
```

Comment:
should probably add a comment here with the justification

Reply:
Added a comment with a question, it would take some refactoring but I'm realizing now these tests will claim that LB passes on this version of the driver. WE know that its just skipped but maybe we should make the effort to properly skip it? I think I can just pass the mocha context down to here.

Reply:
Yes, I think if possible, it would be preferable to properly skip the test, however it may be beyond the scope of this PR - let's file a follow up ticket and add the number here?

Reply:
turned out to be easy enough to do so its now skipping correctly